src/cloud-api-adaptor/podvm/Dockerfile.podvm_builder.rhel

```diff
@@ -10,7 +10,7 @@ FROM registry.access.redhat.com/ubi9/ubi:9.2
 
 ARG ARCH="amd64"
 ARG YQ_ARCH="amd64"
-# PROTOC_ARCH="x86_64" | "s390x_64"
+# PROTOC_ARCH="x86_64" | "s390_64"
```

Comment:
Oh so the PROTOC_ARCH is s390_64. In that case please squash this commit with previous two which deals with the PROTOC_ARCH